drop-transfer/src/ws/server/mod.rs

```diff
@@ -1164,9 +1165,24 @@ fn validate_subpath_for_download(subpath: &FileSubPath) -> crate::Result<()> {
     Ok(())
 }
 
+/// Check file ID for illegal characters so that the temp file is not created in
+/// parent directories
+fn validate_file_id_for_download(file_id: &FileId) -> crate::Result<()> {
+    // Ideally we would allow only alphanumeric characters but we use URL base64
+    // which allows for more
+    const DISALLOWED: &[char] = &['.', '/', '\\', ':'];
```

Comment:
since Base64 consists of alphanumeric chars + 3, maybe it would be better to have a whitelist as opposed to a blacklist for chars?

Reply:
Yeah, that would make sense as well. Lemme change this accordingly

Reply:
Done